packages/itwinui-css/src/bridge.scss

```diff
@@ -252,28 +284,101 @@
   /* ---------------------------------------------------------------------------- */
-  /* ProgressRadial */
+  /* ProgressLinear, ProgressRadial */
 
   // StrataKit progress indicators's default color is not accent
   :is(.iui-progress-indicator-linear, .iui-progress-indicator-radial) {
     --_iui-progress-indicator-color-default-progress: var(--ids-color-icon-neutral-base);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* SideNavigation */
+
+  // iTwinUI background color (--iui-color-background-backdrop)'s global mapping doesn't look correct in this context
+  .iui-side-navigation
+    .iui-sidenav-button:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true']) {
+    background-color: var(--iui-color-background-disabled);
+  }
+
   /* ---------------------------------------------------------------------------- */
   /* Slider */
-  /* TODO: Since Slider is experimental in StrataKit, revisit this when stable on the StrataKit end */
-  /* TODO: Are minLabel and maxLabel fills/colors correct? */
+  /* TODO: Since StrataKit Slider tokens are not added, revisit this when those tokens are added on the StrataKit end */
+
+  // Track color is not --iui-color-border-accent's global mapping
+  .iui-slider {
+    --_iui-slider-track-color: var(--ids-color-bg-accent-base);
+  }
 
   // Background and border is not iui-color-background and iui-color-border-foreground respectively
   .iui-slider-thumb {
-    background-color: var(--ids-color-bg-neutral-faded);
+    background-color: var(--ids-color-bg-neutral-inverse);
     border-color: var(--ids-color-border-neutral-faded);
   }
 
+  /* ---------------------------------------------------------------------------- */
+  /* Surface */
+
+  .iui-surface {
+    // To match the surface similar to StrataKit's dropdown menu
+    background-color: var(--ids-color-bg-elevation-base);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Table */
+
+  // StrataKit table and unselected rows match the backdrop color
+  :is(.iui-table, .iui-table-row:not([aria-selected='true']) .iui-table-cell, .iui-table-paginator, .iui-table-empty) {
+    background-color: var(--iui-color-background-backdrop);
+  }
+
+  // StrataKit table cell border color is muted, unlike iTwinUI's
+  .iui-table-row:not([aria-selected='true']) .iui-table-cell {
+    border-color: var(--ids-color-border-neutral-muted);
+  }
+
+  // Manually undoing iTwinUI case where last selected row has no border-block-end. This is because we overrode
+  // border-color for *all* non-selected rows' cells, even the first non-selected row after the
+  // last selected row that used to provide the border-block-start.
+  /* stylelint-disable-next-line selector-max-compound-selectors */
+  .iui-table-body .iui-table-row[aria-selected='true'] + [aria-selected='true'] .iui-table-cell {
+    border-block-end-color: var(--iui-color-border-accent);
+  }
```

Comment:
This is resulting in vertical borders around the table. Let's leave this out for now. The muted border is not important enough to match.

```suggestion
```